techniques/html/H91.html

```diff
@@ -110,93 +108,94 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
+				<h4>ボタンの事例 B</h4>
+				<p>事例 B では、<code class="language-html">value</code> 属性を用いて "Save"、"Submit"、"Reset" という名前 (name) を指定している。</p>
 <pre xml:space="preserve"><code class="language-html">&lt;input type="button" value="Save"&gt; 
 &lt;input type="submit" value="Submit"&gt;
 &lt;input type="reset" value="Reset"&gt;</code></pre>
 			</section>
 			<section id="buttons-example-c">
-				<h4>Buttons example C</h4>
-				<p>Example C uses the <code class="language-html">alt</code> attribute, in this case "save", as the name.</p>
+				<h4>ボタンの事例 C</h4>
+				<p>事例 C では、<code class="language-html">alt</code> 属性を用いて "save" という名前 (name) を指定している。</p>
 <pre xml:space="preserve"><code class="language-html">&lt;input alt="save" src="save.gif" type="image"&gt;</code></pre>
 			</section>
 			<section id="buttons-example-d">      
-				<h4>Buttons example D</h4>
-				<p>Example D clarifies how the user agent determines the name if the author specifies both the <code class="language-html">alt</code> and <code class="language-html">title</code> attributes of the input element. In this case, the user agent uses the <code class="language-html">alt</code> attribute ("Save") and ignores the <code class="language-html">title</code> attribute.</p>
+				<h4>ボタンの事例 D</h4>
+				<p>事例 D は、コンテンツ制作者が input 要素の <code class="language-html">alt</code> 属性と <code class="language-html">title</code> 属性の両方を指定する場合、ユーザエージェントが名前 (name) をどのように決定するかを明らかにする。この場合、ユーザエージェントは、 <code class="language-html">alt</code> 属性 ("save") を使用して、<code class="language-html">title</code> 属性を無視する。</p>
          
 <pre xml:space="preserve"><code class="language-html">&lt;input alt="save" src="save.gif" title="save the file" type="image"&gt;</code></pre>
 			</section>
 		</section>
       <section class="example" id="example-3-text-input">
-         <h3>Example 3: Text Input</h3>  
+         <h3>事例 3: テキスト入力</h3>  
 			<section id="text-input-example-a">
-				<h4>Text Input example A</h4>
-				<p>In example A, the input field has a role of "editable text". The <code class="language-html">label</code> element is associated to the <code class="language-html">input</code> element via the <code class="language-html">for</code> attribute which references the <code class="language-html">id</code> attribute of the <code class="language-html">input</code> element. The name comes from the <code class="language-html">label</code> element, in this case, "Type of fruit". Its value comes from its value attribute, in this case "bananas".</p>
+				<h4>テキスト入力の事例 A</h4>
+				<p>事例 A では、入力フィールドが「編集可能テキスト」という役割 (role) を持っている。<code class="language-html">label</code> 要素は、<code class="language-html">input</code> 要素の <code class="language-html">id</code> 属性を参照する <code class="language-html">for</code> 属性を介して、<code class="language-html">input</code> 要素に関連付けられている。input 要素の<code class="language-html">label</code> (name) は、<code class="language-html">label</code> 要素で指定した "Type of fruit" となる。値は、value 属性の "bananas" となる。</p>
```

Comment:
> The name comes from the label element, in this case, "Type of fruit".
の訳ですが、「label (name)」は、「名前 (name)」ではないでしょうか？

```suggestion
				<p>事例 A では、入力フィールドが「編集可能テキスト」という役割 (role) を持っている。<code class="language-html">label</code> 要素は、<code class="language-html">input</code> 要素の <code class="language-html">id</code> 属性を参照する <code class="language-html">for</code> 属性を介して、<code class="language-html">input</code> 要素に関連付けられている。input 要素の名前 (name) は、<code class="language-html">label</code> 要素で指定した "Type of fruit" となる。値は、value 属性の "bananas" となる。</p>
```